xilem/examples/http_cats.rs

```diff
@@ -197,12 +198,22 @@ impl Status {
                 .text_size(20.)
                 .alignment(TextAlignment::Middle),
             FlexSpacer::Fixed(10.),
-            image,
-            // TODO: Overlay on top of the image?
-            // HACK: Trailing padding workaround scrollbar covering content
-            // HACK: Bottom padding to workaround https://github.com/linebender/parley/issues/165
-            sized_box(prose("Copyright ©️ https://http.cat").alignment(TextAlignment::End))
-                .padding(Padding::new(0., 15., 10., 0.)),
+            zstack((
+                image,
+                sized_box(
+                    sized_box(
+                        prose("Copyright ©️ https://http.cat")
+                            .line_break_mode(LineBreaking::Clip)
+                            .alignment(TextAlignment::End),
+                    )
+                    .padding(4.)
+                    .rounded(4.)
+                    .background(Color::BLACK.multiply_alpha(0.5)),
+                )
+                // HACK: Trailing padding workaround scrollbar covering content
```

Comment:
I'm not sure this hack comment is entirely correct anymore.

Reply:
I've deleted it